doc/faq/howto_faq.rst

```diff
@@ -494,25 +494,6 @@ you're all done issuing commands and you want to draw the figure now.
    per script, and harmonized the behavior of interactive mode, across most
    backends.
 
-.. _howto-boxplot_violinplot:
```

Comment:
Have you checked that this isn't linked to from anywhere else in the docs?

Reply:
Yes, it's unused.